amical/oifits.py

```diff
@@ -1013,6 +1013,146 @@ def _plot_CP(ax3, l_dic, dic_color, conv_cp, diffWl=False):
     return max_f_cp
 
 
+def _plot_UV_ifu(ax1, fig, l_dic):
+    l_bmax = []
+    all_U, all_V, all_wl = [], [], []
+
+    for dic in l_dic:
+        tmp = _apply_flag(dic)
+        U = tmp.U
+        V = tmp.V
+        wl = tmp.wl
+        l_bmax.append(tmp.bmax)
+        all_U.append(U)
+        all_V.append(V)
+        all_wl.append([wl] * len(U))
+    all_U = np.array(all_U)
+    all_V = np.array(all_V)
+    all_wl = np.array(all_wl) * 1e6
+
+    ax1.scatter(
+        all_U,
+        all_V,
+        s=40,
+        c=all_wl,
+        marker="o",
+        edgecolors="#364f6b",
+        alpha=1,
+        linewidth=0.1,
+        cmap="jet",
+    )
+    sc = ax1.scatter(
+        -all_U,
+        -all_V,
+        s=40,
+        c=all_wl,
+        marker="o",
+        edgecolors="#364f6b",
+        alpha=1,
+        linewidth=0.1,
+        cmap="jet",
+    )
+
+    position = fig.add_axes([0.22, 0.95, 0.1, 0.015])
+    fig.colorbar(sc, cax=position, orientation="horizontal", drawedges=False)
+    ax1.text(
+        0.53, 0.98, r"$\lambda$ [µm]", ha="center", va="center", transform=ax1.transAxes
+    )
+    return l_bmax
+
+
+def _plot_V2_ifu(ax2, l_dic):
+    max_f_vis = []
+    all_V2, all_e_V2, all_freq, all_wl = [], [], [], []
+    for dic in l_dic:
+        tmp = _apply_flag(dic, unit="arcsec")
+        V2 = tmp.vis2
+        e_V2 = tmp.e_vis2
+        sp_freq_vis = tmp.sp_freq_vis
+        max_f_vis.append(np.max(sp_freq_vis))
+
+        all_V2.extend(V2)
+        all_e_V2.extend(e_V2)
+        all_freq.extend(sp_freq_vis)
+        all_wl.extend([tmp.wl] * len(V2))
+
+    all_V2 = np.array(all_V2)
+    all_e_V2 = np.array(all_e_V2)
+    all_freq = np.array(all_freq)
+    all_wl = np.array(all_wl)
+
+    ax2.errorbar(
+        all_freq,
+        all_V2,
+        yerr=all_e_V2,
+        linestyle="None",
```

Comment:
matplotlib has a history of being too flexible with what counts as a None, or what counts as a string, but let's not abuse it
```suggestion
        linestyle=None,
```